addons/vz99/functions/fnc_canSelectNextMagazine.sqf

```diff
@@ -0,0 +1,22 @@
+/*
+ * Author: PabstMirror
+ * 
+ *
+ * Arguments:
+ *
+ * Return Value:
+ * Nothing
+ *
+ * Example:
+ * [] call 
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_magazineClassname"];
+
+private _baseMag = getText (configFile >> "CfgMagazines" >> _magazineClassname >> QGVAR(base));
+TRACE_2("",_magazineClassname,_baseMag);
+
+_baseMag in (magazines ace_player)
```

Comment:
Does looking at the "base" magazine class work for an "in" statement like this? (I mean... I assume so?)